webui/cypress/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## The steps in this Cloud Build script run the frontend e2e test
+
+steps:
+  # Generate API Service client
+  - name: openapitools/openapi-generator-cli:v4.2.3
+    id: Generate Client
+    args: ['generate', '-g','typescript-angular','-i','openapi.yaml','-o','webui/api-client']
+  # Generate User Service client
+  - name: openapitools/openapi-generator-cli:v4.2.3
+    id: Generate User Service Client
+    waitFor: ['-']
+    args: ['generate', '-g','typescript-angular','-i','backend/user-service/user-api.yaml','-o','webui/user-svc-client']
+  # build the frontend Angular app and deploy it to GCS
+  - name: cypress/base
+    id: Prepare
+    entrypoint: npm
+    args: ['install']
+    dir: 'webui'
+  - name: cypress/base
+    waitFor: ['Prepare', 'Generate Client', 'Generate User Service Client']
+    id: Test
+    entrypoint: npm
+    args: ['run', 'e2e', '--', '--config', 'baseUrl=https://${_DOMAIN}']
+    dir: 'webui'
```

Comment:
Is there a missing step to instantiate the credentials template?

Reply:
the initiation is in npm's pre hooks, so no need to run a separate step here.